back/spec/services/xlsx_export/value_visitor_spec.rb

```diff
@@ -109,7 +109,7 @@
         let(:value) { "line 1\nline 2" }
 
         it 'returns nil, because the field is not supported yet' do
```

Comment:
I think we should change the description, as the field is now supported.

Reply:
Sorry I merged this after Amanda's review. I didn't know that the BE changes were on this branch. cc @sebastienhoorens @koendehondt

Reply:
I have applied the change on the `CL-1893-logic` branch after this PR was merged.